### PR TITLE
Couple of styling / bug fixes

### PR DESCRIPTION
**Pull request by Jason2605**
# Summary

- Updates strftime as `System.time()` is the default passed in
- Uses `unwrap()` in Mirgate.run as currently it was just returning the values, so if an Error came through it would crash, but at least with unwrap it should give the reason for failure
- Update some code to use the base query var rather than instantly re-assigning
- Fix bug where `Success()` was being used as a validity check
- Make an if statement a little easier to manage

### CODE REVIEW

**Review by briandowns — APPROVED** [review]
LGTM. Added you to the repo so feel free to merge when ready.

**Comment by Jason2605** [comment]
Think I still need write access, so I'll leave it to you for now

**Pull request merged**
